messenger/message-recorder.rst

```diff
@@ -0,0 +1,69 @@
+.. index::
+    single: Messenger; Record messages
+
+Record Events Produced by a Handler
+===================================
+
+In a example application there is a command (a CQRS message) named ``CreateUser``.
+That command is handled by the ``CreateUserHandler``. The command handler creates
+a ``User`` object, stores that object to a database and dispatches an ``UserCreatedEvent``.
```

Comment:
"a UserCreatedEvent"

Reply:
Are you sure?

Reply:
Almost :) I'm not native. But you wrote "a User" just before. See https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user

Reply:
I would say that "a User" was a typo. 

I read your source. I did not know that. Thank you. I've updated the PR.